### ISSUE TITLE
Persistent Index Error

### ISSUE THREAD

**Issue by Puneeth-Kadamba** [arangodb/arangodb#7165]
ArangoDB Version: 3.3.14
Storage Engine: MMFiles
Deployment Mode: Single Server
Deployment Strategy: ArangoDB Starter \
Configuration: single server
Infrastructure: AWS
Operating System: Ubuntu 16.04
Used Package: Debian or Ubuntu .deb

Long Running Query causing Persistent Index Error.
FATAL unable to initialize RocksDB engine for persistent indexes: IO error: lock /var/lib/arangodb3/rocksdb/LOCK: Resource temporarily unavailable; Maybe your filesystem doesn't provide required features? (Cifs? NFS?)
2018-10-30T09:56:23Z [29093] INFO ArangoDB 3.3.14 [linux] 64bit, using jemalloc, build tags/v3.3.14-0-g377be26, VPack 0.1.30, RocksDB 5.6.0, ICU 58.1, V8 5.7.492.77, OpenSSL 1.0.1f 6 Jan 2014
2018-10-30T09:56:23Z [29093] INFO {authentication} Jwt secret not specified, generating...
2018-10-30T09:56:23Z [29093] INFO detected operating system: Linux version 4.4.0-97-generic (buildd@lcy01-33) (gcc version 5.4.0 20160609 (Ubuntu 5.4.0-6ubuntu1~16.04.4) ) #120-Ubuntu SMP Tue Sep 19 17:28:18 UTC 2017

**Comment by graetzer** [comment]
seems like you got a lock file from a crashed server ? This message can only be logged when you start the server up

**Comment by Puneeth-Kadamba** [comment]
Right. How do I pause the start of the arango service until the lock is released?

**Comment by graetzer** [comment]
You can try to remove the lockfile

**Comment by Puneeth-Kadamba** [comment]
okay thanks. But how do i avoid this from happening again?
